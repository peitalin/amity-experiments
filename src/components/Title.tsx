

import * as React from 'react'
import 'styles/Title.scss'

interface TitleProps {
  className?: string
  style?: Object
}

class Title extends React.Component<TitleProps, any> {
  render() {
    let classNames = this.props.className
      ? ` ${this.props.className}` : ''

    return (
      <div className={'Title' + classNames } style={{ ...this.props.style }}>
        { this.props.children }
      </div>
    )
  }
}


export default Title
