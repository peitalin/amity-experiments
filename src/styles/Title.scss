

div .Title {
  color: #222;
  text-align: center;
  font-size: 32px;
  font-weight: 300;
}
