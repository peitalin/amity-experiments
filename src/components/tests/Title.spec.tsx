


import * as React from 'react'
import * as renderer from 'react-test-renderer'
import { mount, shallow } from 'enzyme'

import Title from '../Title'



test(' component matches snapshot', () => {
  const elR = renderer.create( <Title /> )
  expect(elR.toJSON()).toMatchSnapshot()
})

