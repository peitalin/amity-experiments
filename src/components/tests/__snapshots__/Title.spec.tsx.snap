// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` component matches snapshot 1`] = `
<div
  className="Title"
  style={Object {}}
/>
`;
